_data/changelogs/site-alert.yml

```diff
@@ -19,4 +19,33 @@ changelogURL:
   #   githubPr:
   #   githubRepo:
   #   versionUswds:
-# end changelog item
\ No newline at end of file
+
+items:
+  # 3.1.0 release
+  - date: 2022-10-19
+    summary: Updated the alignment of site alert content to visually match alignment with banner content.
+    summaryAdditional: The alert component was also updated to match banner content alignment.
+    isBreaking: false
+    affectsAccessibility: false
+    affectsAssets: false
+    affectsGuidance: false
+    affectsJavascript: false
+    affectsMarkup: false
+    affectsStyles: true
+    githubPr: 4922
+    githubRepo: uswds
+    versionUswds: 3.1.0
+  # 3.0 release
+  - date: 2022-04-28
+    summary: USWDS 3.0.0 released.
+    summaryAdditional: USWDS 3.0 allows teams to update their projects to modern Sass syntax, improve performance, and reduce the size of project CSS.
+    isBreaking: false
+    affectsAccessibility: true
+    affectsAssets: true
+    affectsGuidance: true
+    affectsJavascript: true
+    affectsMarkup: true
+    affectsStyles: true
+    githubPr: 4656
+    githubRepo: uswds
+    versionUswds: 3.0.0
```

Comment:
I think we can add these items from alert into site alert's changelog:
```
  - date: 2021-03-17
    summary: Restored missing white icons.
    summaryAdditional: Restored <code>error--white</code>, <code>info--white</code>, and <code>warning--white</code> icons.
    affectsAssets: true
    githubPr: 4106
    githubRepo: uswds
    versionUswds: 2.11.1
  - date: 2021-03-17
    summary: Provided better support for color icons that adapt to theme settings.
    summaryAdditional: Now the color of the alert icon will update automatically to adapt to the alert background, or to the color of the alert text.
    affectsAccessibility:
    affectsAssets: true
    affectsStyles: true
    githubPr: 4079
    githubRepo: uswds
    versionUswds: 2.11.0
```

Reply:
Oh, we should include this one as well:
```
  - date: 2021-03-17
    summary: Added the <code>$theme-alert-padding-y</code> setting.
    summaryAdditional:
    affectsAccessibility:
    affectsStyles: true
    githubPr: 4079
    githubRepo: uswds
    versionUswds: 2.11.0
```

Reply:
Updated in f22a39f128051d3b92b067191a47d2092b34425e. For my own reference; I added the missing keys for `affects`.